### PR TITLE
Installation step for macOS/Linux

### PR DESCRIPTION
**Pull request by yamaszone**
Installation step for macOS/Linux

### CODE REVIEW

**Review by wboayue — APPROVED** [review]
Looks good 👍

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
 # `stack` CLI
 Airbus UTM Platform `stack` CLI
+
+## Installation
+
+### macOS/Linux
```
Comment (wboayue) [inline README.md]: Would be more obvious what's happening with we had a separate macOS and linux installation command. Or does catalina report linux?

**Review by sventech — APPROVED** [review]
✔️

File: README.md (hunk 1 above)
Comment (yamaszone) [reply README.md]: I haven't tested on Catalina but it should return `darwin`. I have deliberately made it a one liner to add it easily within pipelines. I have also added OS specific commands for clarity.

**Pull request merged**